Definately still at hack status, this one.  We change the language using langpacks
so we don't want setup refusing to upgrade a user install because he doesn't have the
same language (en_US).  Maybe this could be controlled by a switch to setup.

--- setup2/source/ui/mainevt.cxx.orig	2003-07-22 21:03:01.000000000 +0200
+++ setup2/source/ui/mainevt.cxx	2003-07-22 21:08:19.000000000 +0200
@@ -748,11 +748,13 @@
 
                 eDestInstMode = pCurrentInstall->GetInstallMode();
 
+#if 0
                 //  check if the languages are the same
 				if (nLanguagePos == STRING_NOTFOUND)
 				{
 					m_pEnv->SetUpdateOldVersionWrongLanguage(true);
 				}
+#endif
 
 				// Name and Version are equal
 				m_pEnv->SetUpdateOldVersion();
